templates/_footer.haml

```diff
@@ -8,7 +8,19 @@
     <a href="http://www.gnu.org/licenses/agpl.html">GNU aGPLV3+</a>
   %a{href: "https://github.com/political-memory/political_memory/issues"}
     - trans "Bug tracking system"
+  &nbsp;
   %a{href: "https://github.com/political-memory/political_memory/issues/new"}
     - trans "Report a bug"
   %br
   {% trans "Provided by" %} <a href="http://www.laquadrature.net/">La Quadrature du Net</a>
+  %br
+  %hr
```

Comment:
I'm not that fond of the look with a HR. Maybe get rid of the BR and the HR and put your new links on the same link as the LQDN link ?
Also take care some links seem to include spaces when rendered (ie. an additional underlined space at the end of the link).